core/ir/instr_api.h

```diff
@@ -310,6 +310,13 @@ struct _instr_t {
     byte num_dsts;
     byte num_srcs;
 
+    /** This field assumes values of type #dr_isa_mode_t.
```

Comment:
style: As documented, doxygen comments must either be one line or else the `/**` is on its own line.

Hmm although none of the fields here are doxygen-documented: deliberately b/c we don't want them part of the public interface, b/c we want to change them (like we're doing here).  So maybe just fold this into the other regular comment.

Reply:
Transformed into a regular comment.
I'll remember the syntax for future doxygen comments, thanks!